package.json

```diff
@@ -31,6 +31,7 @@
     "browserstack-runner": "0.3.7",
     "commitplease": "2.0.0",
     "grunt": "0.4.2",
+    "grunt-concurrent": "^2.0.3",
```

Comment:
As you can see on the other modules, we lock their versions, without the `^` prefix. Would you lock this one as well, please?